llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -1643,6 +1646,23 @@ bool SPIRVInstructionSelector::selectLength(Register ResVReg,
       .constrainAllUses(TII, TRI, RBI);
 }
 
+bool SPIRVInstructionSelector::selectDegrees(Register ResVReg,
+                                             const SPIRVType *ResType,
+                                             MachineInstr &I) const {
+
+  assert(I.getNumOperands() == 3);
```

Comment:
Doesn't degrees have 1 operand?

Reply:
Correct, but I believe the first 2 operands of the spir-v instruction are usually used for the `ReturnType` and the `Result` register.